api/app/tests/apps/reporting/test_tasks.py

```diff
@@ -13,3 +13,13 @@ def test_task_save_csv_files_datawarehouse(
         tasks.task_save_csv_files_datawarehouse()
 
         mocked_save_csv_files_datawarehouse.assert_called_once()
+
+
+class TestTaskSaveCSVFilesEndpoint(TestCase):
+
+    @mock.patch('signals.apps.reporting.tasks.save_csv_files_endpoint')
+    def test_task_save_csv_files_endpoint(
+            self, mocked_save_csv_files_endpoint):
```

Comment:
no need to line-break here (we have 120 max length in SIA/Signalen)

Reply:
check